### PR TITLE
2016Q4.support-for-dbc-v6

### PR DESCRIPTION
**Pull request by fgiaimo**
Changes to support the new messages coming to opendlv.core

### CODE REVIEW

File: docker/Makefile [hunk 1]
```diff
@@ -18,7 +18,7 @@
 ###############################################################################
 # Parameters for this Makefile.
 BASE_IMAGE=seresearch/opendlv-core-on-opendavinci-ubuntu-16.04-complete
-BASE_IMAGE_VERSION=v0.4.0
+BASE_IMAGE_VERSION=v0.5.0
```
Comment (chrberger) [inline docker/Makefile]: What further features do we lift to this layer by changing from 0.4.0 to 0.5.0?

**Review by chrberger — COMMENTED** [review]
You changes look fine to me. Please see my question to the change from 0.4.0 to 0.5.0.

File: docker/Makefile (hunk 1 above)
Comment (fgiaimo) [reply docker/Makefile]: mmh by looking at the commit history in .core it looks like the changes were mostly about the Compact Point Cloud and IMU code

**Comment by olbender** [comment]
The FH16 related code should (quite soon) be moved to the opendlv.rhino repository. I suggest that we do that change instead, and we can base that on the Badger repo that is soon ready (they will be very similar). What do you think?

**Review by olbender — APPROVED** [review]
This is fine. But we need to quite soon move all Rhino related code to its own repository.

**Pull request merged**